client/src/pages/Scheduler.js

```diff
@@ -64,95 +67,100 @@ const Scheduler = () => {
         duration: event.duration,
         link: event.link,
       });
+
       setAppointmentDetails({ ...appointmentDetails, eventId: event._id });
+      setEventActive(event.active);
     });
   }, []);
-
-  return (
-    <Paper className={classes.root} elevation={5}>
-      <Grid container direction="row" wrap="nowrap" className={classes.grid}>
-        {appointmentConfirmed ? (
-          <Route to={`${eventDetails.link}/${appointmentConfirmed.id}`}>
-            <Confirmation
-              appointmentDetails={appointmentDetails}
-              eventDetails={eventDetails}
-            />
-          </Route>
-        ) : (
-          <>
-            <Grid item xs={4}>
-              <Overview
-                {...eventDetails}
-                appointmentTime={appointmentDetails.time}
+  if (!eventActive) {
+    return <EventNotActivePage />;
```

Comment:
I'm seeing a flash of the scheduler page before being redirected, maybe you could use an isLoading state to wait until you know the event is active or not

Reply:
We also have this problem with the onboarding page flashing before it detects that you've been onboarded, so whatever solution you end up doing for this we should steal for the onboarding page!

Reply:
@WhoaNellie I corrected this and I think solution is effective and simple. Check it out.